models/token.js

```diff
@@ -17,7 +17,7 @@ const MODEL = {
 
     hash: Joi
         .string()
-        .base64()
+        .regex(/[a-zA-Z0-9_-]+~/)
```

Comment:
I'd have to look it up again, but I thought there were 3 characters that weren't URI safe: `/`, `+`, and `=`. I've seen implementations where the `+` gets converted over to a `.`.

I guess it depends on the base64 library and what does underneath the hood